File: time.py
from datetime import datetime

now = datetime.now()

time_dict = {
    "Year": now.year,
    'Month': now.month,
    'Day': now.day,
    'Hour': now.hour,
    'Minute': now.minute,
    'Second': now.second
}

print(time_dict)

File: time.cs
using System;
using System.Collections.Generic;

class Program
{
    static void Main()
    {
        DateTime now = DateTime.Now;

        Dictionary<string, int> timeDict = new Dictionary<string, int>
        {
            { "Year", now.Year },
            { "Month", now.Month },
            { "Day", now.Day },
            { "Hour", now.Hour },
            { "Minute", now.Minute },
            { "Second", now.Second }
        };

        foreach (var pair in timeDict)
        {
            Console.WriteLine($"{pair.Key}: {pair.Value}");
        }
    }
}
